src/test/java/uk/gov/hmcts/reform/professionalapi/controller/external/OrganisationExternalControllerTest.java

```diff
@@ -202,6 +208,27 @@ public void test_RetrievePaymentAccountByUserEmail() {
         verify(paymentAccountServiceMock, times(1)).findPaymentAccountsByEmail(email);
     }
 
+    @Test
+    public void testRetrievePaymentAccountByUserEmailFromHeader() {
+        final HttpStatus expectedHttpStatus = HttpStatus.OK;
+
+        RequestContextHolder.setRequestAttributes(new ServletRequestAttributes(httpRequest));
+        List<String> authorities = new ArrayList<>();
+        authorities.add(TestConstants.PUI_USER_MANAGER);
+        String email = "test@email.com";
+        when(httpRequest.getHeader(anyString())).thenReturn(email);
+        when(jwtGrantedAuthoritiesConverterMock.getUserInfo()).thenReturn(userInfoMock);
+        when(userInfoMock.getRoles()).thenReturn(authorities);
```

Comment:
please add `verify()` for these `when().thenReturn()`

Reply:
ok.done